demos/using-dev-build/README.md

```diff
@@ -31,6 +31,42 @@ At this point, the replica (for all practical matters, a local version of the In
 
 If the IC actually lets the call (request) through to the `whoami` canister, it means that everything checked out, and the `whoami` canister just responds with the information the IC adds to requests, namely your identity (principal).
 
+### Using the Auth-Client Library To Log In With Internet Identity
+
+DFINITY provides an easy-to-use library to log in with Internet Identity: [agent-js](https://github.com/dfinity/agent-js)
+
+These are the steps required to log in and use the obtained identity for canister calls:
```

Comment:
you mention an "identity" here and below, but it's unclear what that actually is. Two paragraphs above we say that "principals are basically the 'identity' [...]" which might lead the reader to believe that the identity you refer to here is a principal (which I don't think is correct).

Note that I don't think this document is the right place to introduce delegations, unless it can be made really simple to understand.

Reply:
I tried. What do you think?